src/main/java/org/overviewproject/mime_types/MimeTypeDetector.java

```diff
@@ -321,7 +318,6 @@ private boolean isText(byte[] bytes) {
 
     private boolean isAsciiText(byte[] bytes) {
         for (byte b : bytes) {
-            if (b > 0x7f) return false;
```

Comment:
IntelliJ said to remove this IF because `b > 0x7f` always evaluates to false.

Reply:
Maybe this was always buggy? Weird -- I thought it did something. No matter: JUniversalChardet 2.4 isn't buggy so we don't need it any more.